docs/source/documentation.rst

```diff
@@ -268,8 +269,7 @@ By default, AxonDeepSeg installs the CPU version of TensorFlow. To train a model
 using your GPU, you need to uninstall the TensorFlow from your virtual environment, 
 and install the GPU version of it::
 
-    pip uninstall tensorflow
-    pip install tensorflow-gpu==1.3.0
+    pip install tensorflow-gpu
```

Comment:
These instructions don't work, as 1. when tensorflow is still installed, it won't run on the GPU, and 2. this installs the latest version of tensorflow-gpu (e.g. 1.14.0), whereas we need it to be version-matched to what we have for tensorflow in the requirements.txt file.

I just tried the old instructions again on your branch, and did not encounter the error I had before, so I'll just revert back to them for this PR.

Reply:
But I believe if we do that then there is training error ( when `Tensorflow-gpu==1.3.0`). I wasn't able to train using GPU.

Reply:
These instructions which I changed were working when the Tensorflow was still installed. However I agree with your second point ( it installs latest version of `Tensorflow-gpu 1.14.0`). I think this issue can be solved by #225.

Reply:
I just tested your instructions and checked if the GPU was being used (using the `nvidia-smi` command), and they weren't for me. The code would run, but the GPU ressources weren't being used. They are used if I uninstall tensorflow first before I install tensorflow-gpu though.

Reply:
That's strange! I tested my instructions on a new virtual environment, they were working fine. Never mind, I will look into this.